fairscale/nn/data_parallel/sharded_ddp.py

```diff
@@ -69,6 +69,7 @@ def __init__(
         # Communication related attributes
         self.process_group = process_group if process_group is not None else dist.group.WORLD
         self.world_size = dist.get_world_size(self.process_group)
+        self.world_scaling = 1.0 / self.world_size  # > 0
```

Comment:
nit: if self.world_size isn't used anywhere else (only used here AFAICT), should we inline the previous line here?

Reply:
nit: perhaps world_size_scaling instead of world_scaling?